specs/models/DataHubEnvelope.schema.json

```diff
@@ -0,0 +1,137 @@
+{
+  "title": "DataHubEnvelope",
+  "description": "Defines the envelope structure with DHF-specific additions to headers",
+  "type": "object",
+  "properties": {
+    "headers": {
+      "description": "Container for what can typically be considered metadata that pertains to the instance data in the envelope",
+      "type": "object",
+      "properties": {
+        "createdBy": {
+          "description": "MarkLogic user that created this document",
+          "type": "string"
+        },
+        "createdOn": {
+          "description": "dateTime at which this document was created",
+          "type": "string"
+        },
+        "createdUsingFile": {
+          "description": "Will be set when running an ingestion step via DHF and ingesting CSV files",
+          "type": "string"
```

Comment:
we might need datahubCreatedByJob for storing the job ID.

Reply:
That's a metadata key, so it's not part of the document and thus not part of the schema.